vision/getting-started/image_segmentation.ipynb

```diff
@@ -0,0 +1,1003 @@
+        "def call_vertex_image_segmentation(\n",
+        "    image_bytes=None,\n",
+        "    gcs_uri=None,\n",
+        "    mime_type=None,\n",
+        "    mode=\"foreground\",\n",
+        "    prompt=None,\n",
+        "    scribble_bytes=None,\n",
+        "    mask_dilation=None,\n",
+        "    max_predictions=None,\n",
+        "    confidence_threshold=None,\n",
+        "):\n",
+        "    instances = []\n",
+        "\n",
+        "    if image_bytes:\n",
+        "        instances.append(\n",
+        "            {\n",
+        "                \"image\": {\n",
+        "                    \"bytesBase64Encoded\": image_bytes,\n",
+        "                    \"mimeType\": mime_type,\n",
+        "                },\n",
+        "            }\n",
+        "        )\n",
+        "    elif gcs_uri:\n",
+        "        instances.append(\n",
+        "            {\n",
+        "                \"image\": {\"gcsUri\": gcs_uri},\n",
+        "            }\n",
+        "        )\n",
+        "\n",
+        "    if scribble_bytes:\n",
+        "        instances[0][\"scribble\"] = {\n",
+        "            \"image\": {\n",
+        "                \"bytesBase64Encoded\": scribble_bytes,\n",
+        "                \"mimeType\": \"image/png\",\n",
+        "            },\n",
+        "        }\n",
+        "\n",
+        "    if prompt:\n",
+        "        instances[0][\"prompt\"] = prompt\n",
+        "\n",
+        "    parameters = {\"mode\": mode}\n",
+        "\n",
+        "    if mask_dilation:\n",
+        "        parameters[\"maskDilation\"] = mask_dilation\n",
+        "    if max_predictions:\n",
+        "        parameters[\"maxPredictions\"] = max_predictions\n",
+        "    if confidence_threshold:\n",
+        "        parameters[\"confidenceThreshold\"] = confidence_threshold\n",
+        "\n",
+        "    start = timeit.default_timer()\n",
+        "    response = client.predict(\n",
+        "        endpoint=model_endpoint, instances=instances, parameters=parameters\n",
+        "    )\n",
+        "    end = timeit.default_timer()\n",
+        "    print(f\"Vertex Image Segmentation took {end - start:.2f}s.\")\n",
+        "\n",
+        "    return response"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "R45VRKWInfQQ"
+      },
+      "source": [
+        "## Select an image to segment"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "I9caIrZ7Dek1"
+      },
+      "outputs": [],
+      "source": [
+        "# @title Upload an image from local\n",
+        "# @markdown Run this cell to enable and select the `Choose files` button. \\\n",
+        "# @markdown You can then select an image file from your local device to upload. \\\n",
+        "# @markdown Your input image is resized to a maximum dimension of 640 pixels if it is larger.\n",
+        "import base64\n",
```

Comment:
Can you move these descriptors to a markdown cell instead of using the Colab decorators?
It makes them easier to read in other tools.

Reply:
Done.